docs/mkdocs.yml

```diff
@@ -58,8 +58,9 @@ nav:
       - Tutorials:
           - Volume Annotation: webknossos/tutorial_volume_annotation.md
           - Data Sharing: webknossos/tutorial_data_sharing.md
+          - Automation: webknossos/tutorial_automation.md
       - Reference:
-          - webknossos/terminology.md
+          - Terminology: webknossos/terminology.md
```

Comment:
For some reason this showed up as "None" on docs.wk.org 

<img width="292" alt="image" src="https://github.com/scalableminds/webknossos-libs/assets/1105056/c2491916-bcd5-45bb-afb7-27169488d442">